kafka/adminoptions.go

```diff
@@ -302,6 +306,51 @@ func (ao AdminOptionMatchConsumerGroupStates) apply(cOptions *C.rd_kafka_AdminOp
 	return nil
 }
 
+// AdminOptionIncludeAuthorizedOperations decides if the broker should return
+// authorized operations.
+//
+// Default: false
+//
+// Valid for DescribeConsumerGroups, DescribeTopics, DescribeCluster.
+type AdminOptionIncludeAuthorizedOperations struct {
```

Comment:
In case the librdkafka default ever changes, we don't need to change anything here if we use two variables, see similar admin options in the file, too